action.yml

```diff
@@ -20,9 +20,11 @@ runs:
   using: "composite"
   steps: 
     - run: |
-        mkdir -p ~/.aws
-        echo "[${{ inputs.profile-name }}]" >> ~/.aws/credentials
-        echo "aws_access_key_id = $${{ inputs.access-key-id-var }}" >> ~/.aws/credentials
-        echo "aws_secret_access_key = $${{ inputs.secret-access-key-var }}" >> ~/.aws/credentials
-        echo "aws_session_token = $${{ inputs.session-token-var }}" >> ~/.aws/credentials
+        FILE_NAME=${AWS_SHARED_CREDENTIALS_FILE:-$RUNNER_TEMP/.aws/credentials}
```

Comment:
Should we make `$RUNNER_TEMP` an input?

Reply:
It's a default env var supplied by GH Actions. Do you want to be able to override it?

Reply:
Yeah that's what I am thinking so someone can specify a different directory if needed. I'm ok with leaving it until someone requests it since we are the only ones who use it 🤷